apps/smart-contracts/core/deploy/ConfigureStackViaSigner.ts

```diff
@@ -87,49 +60,60 @@ const deployFunction: DeployFunction = async function configureStackViaSigner(
       ],
       bypasslist: undefined,
     },
-    tokenSender: {
-      priceMultiplier: undefined,
-      scaledPriceLowerBound: undefined,
-      allowedMsgSenders: [
-        core.collateral.depositHook.address,
-        core.collateral.withdrawHook.address,
-      ],
+    ppoUSDCtoETHOracle: {
+      observationPeriod: undefined,
+      baseAmount: undefined,
     },
   }
-  console.log('Configuring Collateral via Signer...')
+  console.log('\nConfiguring Collateral via Signer...')
   await core.configureCollateralViaSigner(
     signer,
     deploymentParameters.collateral.depositFee,
     deploymentParameters.collateral.withdrawFee
   )
-  console.log('Configuring DepositHook via Signer...')
+  console.log('\nConfiguring DepositHook via Signer...')
   await core.configureDepositHookViaSigner(
     signer,
     deploymentParameters.depositHook.depositsAllowed,
     deploymentParameters.depositHook.treasury
   )
-  console.log('Configuring WithdrawHook via Signer...')
+  console.log('\nConfiguring DepositHook-TokenSender via Signer...')
+  await configureTokenSenderViaSigner(
+    core.collateral.depositHook.tokenSender,
+    signer,
+    deploymentParameters.depositHook.tokenSender.priceMultiplier,
+    deploymentParameters.depositHook.tokenSender.scaledPriceLowerBound,
+    deploymentParameters.depositHook.tokenSender.allowedMsgSenders
+  )
+  console.log('\nConfiguring WithdrawHook via Signer...')
   await core.configureWithdrawHookViaSigner(
     signer,
     deploymentParameters.withdrawHook.globalPeriodLength,
     deploymentParameters.withdrawHook.globalWithdrawLimitPerPeriod
   )
-  console.log('Configuring DepositRecord via Signer...')
+  console.log('\nConfiguring WithdrawHook-TokenSender via Signer...')
+  await configureTokenSenderViaSigner(
+    core.collateral.withdrawHook.tokenSender,
+    signer,
+    deploymentParameters.withdrawHook.tokenSender.priceMultiplier,
+    deploymentParameters.withdrawHook.tokenSender.scaledPriceLowerBound,
+    deploymentParameters.withdrawHook.tokenSender.allowedMsgSenders
+  )
+  console.log('\nConfiguring DepositRecord via Signer...')
   await core.configureDepositRecordViaSigner(
     signer,
     deploymentParameters.depositRecord.globalNetDepositCap,
     deploymentParameters.depositRecord.userDepositCap,
     deploymentParameters.depositRecord.allowedMsgSenders,
     deploymentParameters.depositRecord.bypasslist
   )
-  console.log('Configuring TokenSender via Signer...')
-  await core.configureTokenSenderViaSigner(
+  console.log('\nConfiguring ETH/USDC Oracle via Signer...')
```

Comment:
nit:
```suggestion
  console.log('\nConfiguring USDC/ETH Oracle via Signer...')
```
since ETH is the quote asset